cdk/src/functions/workmail-organization.is-complete-handler.ts

```diff
@@ -0,0 +1,80 @@
+// eslint-disable-next-line import/no-unresolved
+import {
+  WorkMailClient,
+  DescribeOrganizationCommand,
+  GetMailDomainCommand,
+  UpdateDefaultMailDomainCommand,
+} from '@aws-sdk/client-workmail';
+import { PutParameterCommand, SSMClient } from '@aws-sdk/client-ssm';
+export const PROP_DOMAIN = 'Domain';
+export const PROP_PARAM_NAME = 'PropagationParamName';
+
+const workmail = new WorkMailClient({ region: 'eu-west-1' });
+const SSM = new SSMClient();
+
+export interface isCompleteEvent extends AWSLambda.CloudFormationCustomResourceEventCommon {
```

Comment:
I don't get why you use a custom interface here. 
PhysicalResourceId apparently only exists on update and delete events but not on create event. I think a better way would be to use `CloudFormationCustomResourceEvent` as input to the function and then casting to the right type after checking which event was sent.

Reply:
Ah ok. Now I understand.
Maybe what you want in this case is CdkCustomResourceIsCompleteEvent:

```
// IsComplete events will contain all normal request fields, as well as those returned from
// the initial onEvent handler.
export type CdkCustomResourceIsCompleteEvent = CdkCustomResourceEvent & CdkCustomResourceResponse;```